### PR TITLE
[DEVX-1813] Add launchOrder in saucectl doc

### PR DESCRIPTION
**Pull request by tianfeng92**
Add docs for `launchOrder` config.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by alexplischke — APPROVED** [review]

File: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md [hunk 1]
```diff
@@ -232,7 +232,19 @@ This property is only valid for tests run against emulators. It has no effect on
 sauce:
   visibility: private
 ```
+---
+
+### `launchOrder`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Control starting order of suites. If unspecified or empty, the default is the order in which suites are written in the config file.
```
Comment (lysannep) [inline docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:240]:
```suggestion
Specifies the execution order for your test suites. When set to `fail rate`, test suites with the highest failure rate will execute first. If unspecified, test suites will execute in the order in which they are written in the configuration file. 
```

File: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md [hunk 2]
```diff
@@ -232,7 +232,19 @@ This property is only valid for tests run against emulators. It has no effect on
...
 
+* `fail rate`: Suites that historically have the highest failure rate start first.
```
Comment (lysannep) [inline docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:242]:
```suggestion
```

File: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md (hunk 1 above)
Comment (lysannep) [reply docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:240]: @tianfeng92 I added a couple suggested changes to merge the two lines into one -- the suggestions apply to all files.

Comment (tianfeng92) [reply docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:240]: 🙆‍♀️

**Review by lysannep — APPROVED** [review]

File: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md (hunk 1 above)
Comment (lysannep) [reply docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md]: @tianfeng92 Thank you!

**Pull request merged**